app/views/GermlineView/components/Board/components/ParamsContext/index.ts

```diff
@@ -16,6 +18,8 @@ const ParamsContext = React.createContext({
   setOffset: (offset: number) => {},
   reviewFilter: false,
   setReviewFilter: (reviewFilter: boolean) => {},
+  searchText: '',
+  setSearchText: (searchText: string) => {},
 });
```

Comment:
can do `React.createContext<ParamsContextType>` 

to avoid having to re-declare your types again

Reply:
Oh very cool, I didn't know about that